server/controllers/comment.controller.js

```diff
@@ -22,9 +27,16 @@ class CommentController {
     const { id } = req.user;
     const { slug } = req.params;
```

Comment:
Hi @TheDrizzyWay please a little explanation on the use of 'slug'

Reply:
We are using the article slug based on the endpoint specs in the repository's readme file. It states that the article slug should be a part of the endpoint and be used to reference the article.